projects/storefrontlib/src/cms-components/product/product-interests/stock-notification/notification-dialog/notification-dialog.component.spec.ts

```diff
@@ -59,61 +59,53 @@ describe('NotificationDialogComponent', () => {
   beforeEach(() => {
     fixture = TestBed.createComponent(NotificationDialogComponent);
     component = fixture.componentInstance;
+    component.subscribeSuccess$ = of(true);
     fixture.detectChanges();
   });
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
 
-  it('should show selected channels when selectedChannels are exist', () => {
-    selectedChannels = [
-      {
-        channel: 'EMAIL',
-        value: 'test@sap.com',
-      },
-      {
-        channel: 'SMS',
-        value: '13800000831',
-      },
-    ];
-    component.subscribeSuccess$ = of(true);
-    component.selectedChannels = selectedChannels;
+  it('should show notification dialog', () => {
     fixture.detectChanges();
```

Comment:
if you use 'fixture.detectChanges();' in  beforeEach segment, you do not need to call it in this place